dnsrecon.py

```diff
@@ -359,7 +359,7 @@ def brute_srv(res, domain, verbose=False, thread_num=None):
                                          "target": rcd[2],
                                          "address": rcd[3],
                                          "port": rcd[4]})
-                print_good(f"     {rcd[0]} {rcd[1]} {rcd[2]} {rcd[3]} {rcd[4]}")
+                print_good(f'     {rcd[0]} {rcd[1]} {rcd[2]} {rcd[3]} {rcd[4]}')
```

Comment:
Are you meant to have a massive space here, if it is for formatting when printing you will want to use /t to add that instead of the spaces.

Reply:
Yes this was intentional.